apps/worker/src/config/env.validators.ts

```diff
@@ -54,6 +54,7 @@ export const envValidators = {
   MONGO_URL: str(),
   MONGO_MIN_POOL_SIZE: num({ default: 10 }),
   MONGO_MAX_POOL_SIZE: num({ default: 500 }),
```

Comment:
Replacing with our new lower default

```suggestion
  MONGO_MAX_POOL_SIZE: num({ default: 50 }),
```